action.yaml

```diff
@@ -95,13 +95,20 @@ runs:
           fi
         }
 
+        error=$(git remote set-url origin "${{ inputs.gerrit-url }}/${{ inputs.gerrit-project }}")
+        if [ $exit_code -ne 0 ]
```

Comment:
You do not set exit_code so the code would fail with `line 29: [: -ne: unary operator expected`.

https://github.com/vrpolakatcisco/.github/actions/runs/16653525916/job/47132378367#step:2:135